ncs/sign_script.py

```diff
@@ -68,30 +48,32 @@ class SuitIds(Enum):
     SUIT_MANIFEST_COMPONENT_ID = 5
 
 
-DEFAULT_KEY_ID = 0x40000000
-
-KEY_IDS = {
-    "nRF54H20_sample_root": 0x4000AA00,  # MANIFEST_PUBKEY_OEM_ROOT_GEN1
-    "nRF54H20_sample_app": 0x40022100,  # MANIFEST_PUBKEY_APPLICATION_GEN1
-    "nRF54H20_sample_rad": 0x40032100,  # MANIFEST_PUBKEY_RADIO_GEN1
-}
-
-DOMAIN_NAME = "nordicsemi.com"
-
-
 class SignerError(Exception):
     """Signer exception."""
 
 
-class Signer:
+def _import_module_from_path(module_name, file_path):
+    # Helper function to import a python module from a file path.
```

Comment:
nit: This comment could be added as a docstring.

Reply:
Fixed